File: Task 5/test3 UIButton/test3 UIButton/ViewController.m
//
//  ViewController.m
//  test3 UIButton
//
//  Created by 张皓 on 4/22/16.
//  Copyright © 2016 张皓. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()
@property (weak, nonatomic) IBOutlet UIButton *customButton;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self setUpButton];
}

- (void) setUpButton {
    UIButton * btn = [[UIButton alloc] initWithFrame:CGRectMake(0, 0, 200, 100)];
    btn.backgroundColor = [UIColor redColor];
    
    [self.view addSubview:btn];
    
    [btn setImage:[UIImage imageNamed:@"1.jpg"] forState:UIControlStateNormal];
    
    [btn setImage:[UIImage imageNamed:@"2.jpg"]  forState:UIControlStateHighlighted];
    
    [btn setTitle:@"normal" forState:UIControlStateNormal];
    
    [btn setTitle:@"hightlighted" forState:UIControlStateHighlighted];

    UIView
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)stateButton:(UIButton *)sender {
    switch (sender.tag) {
        case 1:
            [self.customButton setSelected:YES];
            [self.customButton setEnabled:YES];
            break;
        case 2:
            [self.customButton setEnabled:NO];
            [self.customButton setSelected:NO];
            break;
        default:
            [self.customButton setEnabled:YES];
            [self.customButton setSelected:NO];
            break;
    }
}

@end

File: task 7/scrollviewd动态改变图片/scrollviewd动态改变图片/ViewController.swift
//
//  ViewController.swift
//  scrollviewd动态改变图片
//
//  Created by 张皓 on 16/5/18.
//  Copyright © 2016年 张皓. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    var banner: Banner!
    var auto: Bool!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        banner = Banner(frame: self.view.frame)
        self.view.addSubview(banner)
        
        //控制是否自动轮播及是否添加pagecontrol，
//        auto = false
//        if auto! {
//            banner.pageControlEnabled = false
//            banner.autoScroll = false
//        }
        
        

        banner.setBannerImages(["1.jpg","2.jpg","3.jpg","4.jpg","5.jpg","6.jpg"])
        banner.autoScrollTimeInterval = 5

        
        
        
        
      
        
        
    }
    
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    
}

